src/arc.ts

```diff
@@ -201,7 +190,7 @@ export async function initializeArc(): Promise<Arc> {
   console.log(arcSettings);
   console.log(`alchemy-server (process.env.API_URL): ${process.env.API_URL}`);
   if (arcSettings.web3Provider.isMetaMask) {
-    console.log("Using Metamask Web3 provider");
+    console.log(`Using ${arcSettings.web3Provider.isSafe ? "Gnosis Safe" : "Metamask"} Web3 provider`);
```

Comment:
so the provider of Gnosis Safe returns `isMetamask` as true?

Reply:
Yes, it returns both flags set to true